src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -1,22 +1,26 @@
 package org.broadinstitute.dsde.workbench.leonardo.dao
 
-import com.google.api.services.dataproc.model._
-import org.broadinstitute.dsde.workbench.leonardo.config.DataprocConfig
-import org.broadinstitute.dsde.workbench.google.GoogleUtilities
 import akka.actor.ActorSystem
 import akka.http.scaladsl.model.StatusCodes
+import cats.data.OptionT
+import cats.instances.future._
 import com.google.api.client.auth.oauth2.Credential
 import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
 import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport
 import com.google.api.client.googleapis.json.GoogleJsonResponseException
 import com.google.api.client.json.jackson2.JacksonFactory
+import com.google.api.services.compute.Compute
+import com.google.api.services.compute.model.Instance
 import com.google.api.services.dataproc.Dataproc
+import com.google.api.services.dataproc.model.{ClusterStatus => _, _}
```

Comment:
??!

Reply:
Import everything from `com.google.api.services.dataproc.model` _except_ `ClusterStatus`.

I tried `import org.broadinstitute.dsde.workbench.leonardo.model.{ClusterStatus => LeoClusterStatus}` but ran into some strange issues with Scala enum's. I can try again.

Reply:
Figured it out, I need to not only rename the `ClusterStatus` object, but the inner `type ClusterStatus = Value`. Scala enums are weird..